### ISSUE TITLE
[CI] BucketScriptIT classMethod failing

### ISSUE THREAD

**Issue by bpintea** [elastic/elasticsearch#86293]
**Build scan:**
https://gradle-enterprise.elastic.co/s/kvmia7oqk6hro/tests/:server:internalClusterTest/org.elasticsearch.search.aggregations.pipeline.BucketScriptIT/classMethod

**Reproduction line:**
`null`

**Applicable branches:**
master

**Reproduces locally?:**
Didn't try

**Failure history:**
https://gradle-enterprise.elastic.co/scans/tests?tests.container=org.elasticsearch.search.aggregations.pipeline.BucketScriptIT&tests.test=classMethod

**Failure excerpt:**
```
java.lang.AssertionError: {127.0.0.1:41039}{3sIv0v__T_-pW5k0AAAAAA}{127.0.0.1}{127.0.0.1:41039} vs {node_s1}{No5FsVP2Qkq3-PKqk14q5g}{Atqi1t9JRG6B_8D0jnfoDg}{127.0.0.1}{127.0.0.1:34367}{ilmrt}

  at __randomizedtesting.SeedInfo.seed([8CD153B8EF136E0E]:0)
  at org.elasticsearch.transport.TransportService.doStop(TransportService.java:305)
  at org.elasticsearch.common.component.AbstractLifecycleComponent.stop(AbstractLifecycleComponent.java:68)
  at org.elasticsearch.node.Node.stop(Node.java:1301)
  at org.elasticsearch.node.Node.close(Node.java:1320)
  at org.elasticsearch.test.InternalTestCluster$NodeAndClient.close(InternalTestCluster.java:1054)
  at org.elasticsearch.core.IOUtils.close(IOUtils.java:73)
  at org.elasticsearch.core.IOUtils.close(IOUtils.java:115)
  at org.elasticsearch.core.IOUtils.close(IOUtils.java:98)
  at org.elasticsearch.test.InternalTestCluster.close(InternalTestCluster.java:905)
  at org.elasticsearch.test.ESIntegTestCase.afterInternal(ESIntegTestCase.java:575)
  at org.elasticsearch.test.ESIntegTestCase.afterClass(ESIntegTestCase.java:2225)
  at jdk.internal.reflect.GeneratedMethodAccessor42.invoke(null:-1)
  at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  at java.lang.reflect.Method.invoke(Method.java:568)
  at com.carrotsearch.randomizedtesting.RandomizedRunner.invoke(RandomizedRunner.java:1758)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$7.evaluate(RandomizedRunner.java:909)
  at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.tests.util.TestRuleStoreClassName$1.evaluate(TestRuleStoreClassName.java:38)
  at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.tests.util.TestRuleAssertionsRequired$1.evaluate(TestRuleAssertionsRequired.java:53)
  at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
  at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
  at org.apache.lucene.tests.util.TestRuleIgnoreTestSuites$1.evaluate(TestRuleIgnoreTestSuites.java:47)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:375)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl.lambda$forkTimeoutingTask$0(ThreadLeakControl.java:831)
  at java.lang.Thread.run(Thread.java:833)

```

**Comment by DaveCTurner** [comment]
This should be fixed by #86315.

**Comment by Luegg** [comment]
@DaveCTurner this looks like another instance of the same issue: https://gradle-enterprise.elastic.co/s/qnsqd3jakk53u/tests/:server:internalClusterTest/org.elasticsearch.action.admin.indices.rollover.RolloverIT/testRolloverWithNewIndexName?anchor=eyJvdXRwdXRMaW5lIjoiMC0zODYifQ&expanded-stacktrace=WyIwIl0

**Comment by DaveCTurner** [comment]
Thanks @Luegg for the ping, you're right the fix only went into 8.3. I've opened #86843 to just remove the failing assertion in 8.2 since it's not hugely important here.
